### ISSUE TITLE
Empty inventory require to join the game

### ISSUE THREAD

**Issue by andris155** [ShaneBeeStudios/HungerGames#82]
**Is your feature request related to a problem? Please describe:**

The items from players inventory sometimes lost when joined with not empty inventory to game.

**Describe the solution you'd like:**

Please add a config option for empty inventory require to join the game.

**Comment by ShaneBeee** [comment]
The inventory is saved when the player joins the game, and when they exit it is given back to them.
If that is not working for you, you may have a conflicting plugin.

Do you have any plugins that manage inventories?

**Comment by andris155** [comment]
Yes, the RestirectedCreative plugin store creative inventory-s separately on my server.

**Comment by andris155** [comment]
Maybe cause inventory lost when we restart the server with players playing hungergames.

**Comment by ShaneBeee** [comment]
I recommend stopping all games before restarting your server, just to be safe.

**Comment by andris155** [comment]
If server crash we can't stop the games before restart.

**Comment by ShaneBeee** [comment]
Unfortunately there is nothing I can do in terms of your server crashing.

**Comment by andris155** [comment]
If players can enter with an only empty inventory, there is nothing to lose.

**Comment by ShaneBeee** [comment]
That is true. I will look at adding an option to require an empty inventory.

**Comment by andris155** [comment]
Thanks.
